### PR TITLE
NoBug - Change entitlement path

### PR DESCRIPTION
**Pull request by isabelrios**
This PR is to check that everything keeps compiling and working despite this change

* Not sure why the two first changes are done when changing the path, the line is removed and although it is added it is still marked as a difference. I will not land this until that is clear.

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5163,7 +5163,7 @@
 				EB9A179B20E69A7F00B12184 /* ThemeManager.swift in Sources */,
 				EBFDB790211C83A5005CCA2F /* BrowserViewController+FindInPage.swift in Sources */,
 				EBB89509219398E500EB91A0 /* TabContentBlocker+ContentScript.swift in Sources */,
-				0B3E7D951B27A7CE00E2E84D /* AboutHomeHandler.swift in Sources */,
+                               0B3E7D951B27A7CE00E2E84D /* AboutHomeHandler.swift in Sources */,
```
Comment (isabelrios) [inline Client.xcodeproj/project.pbxproj]: Looks like there is an extra space although when editing the file it showed the same alignment...will modify it

Comment (st3fan) [reply Client.xcodeproj/project.pbxproj]: @isabelrios 
I'm not sure about this .. i thought these entitlements had no prefix at all, but if they already had `$(SRCROOT)` then that should be turned into an absolute path?

(I don't know if `$(PROJECT_ROOT)` is actually a thing - i was just using that as an example)

File: Client.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5826,7 +5826,7 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
 				APPLICATION_EXTENSION_API_ONLY = YES;
-				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Extensions/Entitlements/Fennec.entitlements";
+				CODE_SIGN_ENTITLEMENTS = "$(inherited)Extensions/Entitlements/Fennec.entitlements";
```
Comment (isabelrios) [inline Client.xcodeproj/project.pbxproj]: @st3fan, with this `$(inherited)` change I passed the error we were having, now there are issues with the Codesigns `[Failed to find Codesign Groups](https://app.bitrise.io/build/f179616a38e26400)` 
Investigating...

**Comment by isabelrios** [comment]
Removing the r? until everything works fine there

**Comment by isabelrios** [comment]
Too many changes in this PR in the schemes and so to have the tests running on Bitrise, so this PR will be open for a little bit more in case we need to do some more testing, but the official one to cover this change is PR #4487

**Comment by isabelrios** [comment]
Closing this PR because the changes already landed on master on PR #4487

**Pull request closed without merging**